File: app/src/main/java/com/notabarista/authentication/repository/UserRepository.java
package com.notabarista.authentication.repository;

import com.notabarista.authentication.model.User;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface UserRepository extends JpaRepository<User, Integer> {
    public User findByEmail(String Email);

}

File: app/src/main/kotlin/org/notabarista/app/model/UserRepository.kt
package org.notabarista.app.model

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository

//  Since we’ve extended UserRepository from JpaRepository interface, all the CRUD methods on User entity is readily available to us
@Repository
interface  UserRepository : JpaRepository<User,Long>

